tensorflow/core/kernels/check_numerics_op_gpu.cu.cc

```diff
@@ -67,9 +67,9 @@ struct CheckNumericsLaunch {
         (d.getNumCudaMultiProcessors() * d.maxCudaThreadsPerMultiProcessor()) /
```

Comment:
`#if GOOGLE_CUDA` section is incorrect now, and yet again these 2 sections could possibly be merged together.

Reply:
you are right...will combine the two